src/weatherIconLoader.js

```diff
@@ -0,0 +1,52 @@
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
This is where all the fetching of the data and changing the html contents of the weather div happens, it's pretty straight forward and readable so I don't think it takes much documentation to understand